codechecker_lib/arg_handler.py

```diff
@@ -433,14 +428,17 @@ def handle_plist(args):
 
     try:
         items = [(plist, args, context) for plist in os.listdir(args.directory)]
-        pool.map_async(consume_plist, items, 1).get(float('inf'))
+        pool.map_async(consume_plist, items, 1)
```

Comment:
Maybe not giving any argument would be even better? 
https://docs.python.org/2/library/multiprocessing.html#multiprocessing.pool.AsyncResult.get